lms/models/lti_user.py

```diff
@@ -11,6 +13,9 @@ class LTIUser:
     """The user_id LTI launch parameter."""
 
     roles: str
```

Comment:
Renaming this will take a bit of extra effort as there are in flight tokens with the current name.

We could tackled that in a follow up.